### PR TITLE
5696: Isapi db healthcheck

### PR DESCRIPTION
**Pull request by assoomro**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs [hunk 1]
```diff
@@ -131,6 +136,15 @@ public void ConfigureApp(IApplicationBuilder app)
             {
                 endpoints.MapDefaultControllerRoute();
                 endpoints.MapControllers();
+                endpoints.MapHealthChecks("/health/live", new HealthCheckOptions
+                {
+                    Predicate = (healthCheckRegistration) => healthCheckRegistration.Tags.Contains(HealthCheckTags.Live)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs:141]: The brackets around the lambda parameter can be omitted here.

File: src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs [hunk 2]
```diff
@@ -131,6 +136,15 @@ public void ConfigureApp(IApplicationBuilder app)
...
+                });
+
+                endpoints.MapHealthChecks("/health/ready", new HealthCheckOptions
+                {
+                    Predicate = (healthCheckRegistration) => healthCheckRegistration.Tags.Contains(HealthCheckTags.Ready)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs:146]: The brackets around the lambda parameter can be omitted here.

File: src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs (hunk 1 above)
Comment (assoomro) [reply src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs:141]: 🤦‍♂ D'oh! I'll get rid of them.

File: src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs (hunk 2 above)
Comment (assoomro) [reply src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs:146]: 🤦‍♂ D'oh! I'll get rid of them

**Pull request merged**